doc/classes/Texture2D.xml

```diff
@@ -123,7 +123,7 @@
 			<return type="Image" />
 			<description>
 				Returns an [Image] that is a copy of data from this [Texture2D] (a new [Image] is created each time). [Image]s can be accessed and manipulated directly.
-				[b]Note:[/b] This will fetch the texture data from the GPU, which might cause performance problems when overused.
+				[b]Note:[/b] It will return an empty image if this [Texture2D] is invalid. This will fetch the texture data from the GPU, which might cause performance problems when overused.
```

Comment:
```suggestion
				[b]Note:[/b] Thid will return an empty image if this [Texture2D] is invalid.
				[b]Note:[/b] This will fetch the texture data from the GPU, which might cause performance problems when overused.
```
Generally afaik we use this wording, and I'll make this two notes as they aren't related

Reply:
Thanks!

Reply:
It returns a `Ref<Image>()`, can this be considered as null ? print it will show a `"<Object#null>"` , it does have a type lol.